package exception1;

public class Throw_Ex2 {
	public void test() throws ArithmeticException
	{
		Throw_Ex2_Test t=new Throw_Ex2_Test();
		t.input();
	}

}
